### PR TITLE
Fix `Exec::start` to respect the TTY settings of the exec instance

### PR DESCRIPTION
**Pull request by vv9k**

### CODE REVIEW

File: src/api/containers.rs [hunk 1]
```diff
@@ -522,7 +522,14 @@ impl Container {
         self.podman
             .post_json(&ep, Payload::Json(opts.serialize()?), Headers::none())
             .await
-            .map(|resp: models::IdResponse| Exec::new(self.podman.clone(), resp.id))
+            .map(|resp: models::IdResponse| {
+                let is_tty = opts.params.get("tty").and_then(|v| v.as_bool()).unwrap_or_default();
```
Comment (marhkb) [inline src/api/containers.rs:526]: Everything works fine if the key starts with a capital letter. 
```suggestion
                let is_tty = opts.params.get("Tty").and_then(|v| v.as_bool()).unwrap_or_default();
```

**Comment by marhkb** [comment]
This works perfectly fine now.

File: src/api/containers.rs (hunk 1 above)
Comment (vv9k) [reply src/api/containers.rs]: Fixed, thanks!

**Comment by vv9k** [comment]
Thank you for the feedback. I'll fix the clippy lints on master, rebase this branch and probably merge soon.

**Comment by vv9k** [comment]
Ran tests locally and everything seems fine. There is a problem with CI currently but I'll resolve it in the master branch after merge.

**Pull request merged**